### PR TITLE
Sliding window maximum

### PR DESCRIPTION
**Pull request by Mugdha-Hazra**

### CODE REVIEW

**Comment by Mugdha-Hazra** [comment]
Hello @rish-singhal @Rohan27s  sir, added code for Sliding Window Maximum #1750 can you please review it?

File: Code/C++/Sliding_Window_Maximum.cpp [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    cout << "\nThe resultant array : ";
+    for (auto i = 0; i < r.size(); ++i)
+    {
+        cout << r[i] << " ";
+    }
+    return 0;
+}
+/*
+           Time complexity is O(N).
+
```
Comment (Rohan27s) [inline Code/C++/Sliding_Window_Maximum.cpp:53]: Mention space complexity also

**Review by Rohan27s — CHANGES_REQUESTED** [review]

File: Code/C++/Sliding_Window_Maximum.cpp (hunk 1 above)
Comment (Mugdha-Hazra) [reply Code/C++/Sliding_Window_Maximum.cpp:54]: @Rohan27s sir done.

**Review by Rohan27s — APPROVED** [review]

**Comment by Mugdha-Hazra** [comment]
thank you, @Rohan27s sir!!

**Pull request merged**